spring-mvc/src/main/java/cz/fi/muni/pa165/springmvc/controllers/LoginController.java

```diff
@@ -32,12 +32,12 @@
     @Autowired
     private MessageSource messageSource;
 
-    @RequestMapping(value = "/login", method = RequestMethod.GET)
+    /*@RequestMapping(value = "/login", method = RequestMethod.GET)
     public String login() {
         return "login";
     }
 
-    @RequestMapping(value = "/login", method = RequestMethod.POST)
+    /*@RequestMapping(value = "/login", method = RequestMethod.POST)
     public String login(@RequestParam String email, @RequestParam String password, ServletRequest r, Model model, Locale locale) {
```

Comment:
i know about these, i decided not to drop that code entirely now, just comment it out